casr/tests/tests.rs

```diff
@@ -3647,7 +3647,7 @@ fn test_casr_libfuzzer() {
         .parent()
         .unwrap();
     let mut cmd = Command::new(*EXE_CASR_LIBFUZZER.read().unwrap());
-    cmd.args(["-i", &paths[0], "-o", &paths[1], "--", &paths[2]])
+    cmd.args(["-i", &paths[0], "-o", &paths[1], "--", &paths[2], "@@"])
```

Comment:
Do we need to put @@? Maybe handle it inside `casr-libfuzzer`. For now it breaks backward compatibility.

Reply:
removed